File: do-connect-capstone-project/src/main/java/com/cogent/requests/UserRequest.java
package com.cogent.requests;

import java.util.ArrayList;
import com.cogent.entity.SimpleGrantedAuthority;

import lombok.Data;
@Data
public class UserRequest {
	private int id;
	private String username;
	private String password;
	private String name;
	private String email;
	SimpleGrantedAuthority authorities;
}

File: do-connect/src/app/userrequest.ts
export class UserRequest{
    id:number;
    username:string;
    password:string;
    name:string;
    email:string;
    authorities:any[];
    constructor(id:number=0, username:string="", password:string="", name:string="",email:string="",authorities:any[]){
        this.id=id;
        this.username=username;
        this.password=password;
        this.name=name;
        this.email=email;
        this.authorities=authorities;
    }
}